setup.py

```diff
@@ -3,7 +3,7 @@
 # Stops exit traceback on tests
 # TODO this makes flake8's F401 fail - maybe there's a better way
 try:
-    import multiprocessing
+    import multiprocessing # noqa
```

Comment:
Does `# flake8: noqa` work here? If so, I think it's better because it's more obvious what it pertains to